src/integrationTest/java/uk/nhs/hee/tis/trainee/forms/api/LtftResourceIntegrationTest.java

```diff
@@ -352,4 +357,51 @@ void shouldUpdateLtftFormForTrainee() throws Exception {
     assertThat("Unexpected saved record id.", savedRecords.get(0).getId(),
         is(formSaved.getId()));
   }
+
+  @Test
+  void shouldReturnNotFoundWhenDeletingNonExistentLtftForm() throws Exception {
+    String token = TestJwtUtil.generateTokenForTisId(TRAINEE_ID);
+    mockMvc.perform(delete("/api/ltft/" + UUID.randomUUID())
+            .header(HttpHeaders.AUTHORIZATION, token))
+        .andExpect(status().isNotFound())
+        .andExpect(jsonPath("$").doesNotExist());
+  }
+
+  @ParameterizedTest
+  @EnumSource(value = LifecycleState.class, names = {"DRAFT"}, mode = EnumSource.Mode.EXCLUDE)
+  void shouldReturnBadRequestWhenServiceCantDeleteLtftForm(LifecycleState lifecycleState)
+      throws Exception {
+    LtftForm form = new LtftForm();
+    form.setTraineeTisId(TRAINEE_ID);
+    AbstractAuditedForm.Status.StatusInfo statusInfo
+        = AbstractAuditedForm.Status.StatusInfo.builder().state(lifecycleState).build();
+    form.setStatus(new AbstractAuditedForm.Status(statusInfo, List.of(statusInfo)));
+    LtftForm formSaved = template.save(form);
+
+    UUID savedId = formSaved.getId();
+    String token = TestJwtUtil.generateTokenForTisId(TRAINEE_ID);
+    mockMvc.perform(delete("/api/ltft/" + savedId)
+            .header(HttpHeaders.AUTHORIZATION, token))
+        .andExpect(status().isBadRequest())
+        .andExpect(jsonPath("$").doesNotExist());
+  }
+
+  @Test
+  void shouldDeleteLtftForm() throws Exception {
+    LtftForm form = new LtftForm();
+    form.setTraineeTisId(TRAINEE_ID);
+    AbstractAuditedForm.Status.StatusInfo statusInfo
+        = AbstractAuditedForm.Status.StatusInfo.builder().state(LifecycleState.DRAFT).build();
+    form.setStatus(new AbstractAuditedForm.Status(statusInfo, List.of(statusInfo)));
```

Comment:
This (and similar lines) will get cleaned-up when we have a setLifecycleState() method.